src/server/game/Spells/SpellMgr.cpp

```diff
@@ -3088,6 +3089,12 @@ void SpellMgr::LoadSpellInfoCorrections()
         spellInfo->DmgClass = SPELL_DAMAGE_CLASS_MAGIC;
     });
 
+    // Frenzied Regeneration
+    ApplySpellFix({ 22845 }, [](SpellInfo* spellInfo)
+    {
+        spellInfo->SchoolMask = SPELL_SCHOOL_MASK_NATURE;
```

Comment:
[22:15:59] <Shauren> afaik school is sent in combat log packets so that might not be valid

Reply:
`SPELL_SCHOOL_MASK_NORMAL` makes critical chance to be zero according to [`Unit::SpellCritChanceDone`](https://github.com/TrinityCore/TrinityCore/blob/3.3.5/src/server/game/Entities/Unit/Unit.cpp#L6940-L6941). Adding an `if` here could be a solution?